web/packages/teleterm/src/ui/Vnet/DocumentVnetDiagReport.tsx

```diff
@@ -0,0 +1,301 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+
+import { Button, Alert as DesignAlert, Flex, H1, Link, Stack } from 'design';
+import { AlertProps } from 'design/Alert/Alert';
+import Table, { TextCell } from 'design/DataTable';
+import { displayDateTime } from 'design/datetime';
+import {
+  Copy,
+  Download,
+  Refresh,
+  Check as SuccessIcon,
+  Warning as WarningIcon,
+} from 'design/Icon';
+import { P1, P2 } from 'design/Text/Text';
+import { HoverTooltip } from 'design/Tooltip';
+import { Timestamp } from 'gen-proto-ts/google/protobuf/timestamp_pb';
+import * as diag from 'gen-proto-ts/teleport/lib/vnet/diag/v1/diag_pb';
+import { pluralize } from 'shared/utils/text';
+
+import { reportOneOfIsRouteConflictReport } from 'teleterm/helpers';
+import Document from 'teleterm/ui/Document';
+import type * as docTypes from 'teleterm/ui/services/workspacesService';
+
+export function DocumentVnetDiagReport(props: {
+  visible: boolean;
+  doc: docTypes.DocumentVnetDiagReport;
+}) {
+  const { report } = props.doc;
+  const { networkStackAttempt } = report;
+  const { networkStack } = networkStackAttempt;
+  const createdAt = displayDateTime(Timestamp.toDate(report.createdAt));
+
+  return (
+    <Document visible={props.visible}>
+      <Stack
+        gap={4}
+        maxWidth="680px"
+        width="100%"
+        mx="auto"
+        mt={4}
+        p={5}
+        backgroundColor="levels.surface"
+        borderRadius={2}
```

Comment:
[I asked the design team](https://gravitational.slack.com/archives/C07KG2RFA/p1739871197453609) if this should have a `levels.surface` background.

Reply:
Merging with the background color on for now.